axlearn/cloud/gcp/jobs/bastion_vm.py

```diff
@@ -335,7 +341,8 @@ def _execute(self):
             "\nView bastion outputs with:\n"
             f"gsutil cat {os.path.join(self.bastion_dir, 'logs', cfg.job_name)}\n"
             "\nCheck job history with:\n"
-            f"{infer_cli_name()} gcp bastion history --name={cfg.name} --job_name={cfg.job_name}"
+            f"axlearn gcp bastion history --name={cfg.name} "
```

Comment:
Was the `infer_cli_name()` -> `axlearn` change necessary for some reason?

Reply:
Yes, I noticed that it doesn't work for `ajax gcp bastion history ...`.